package.json

```diff
@@ -22,6 +23,7 @@
     "react-dom": "18.0.0",
     "react-native": "0.69.5",
     "react-native-gesture-handler": "~2.5.0",
+    "react-native-gifted-chat": "^1.0.4",
     "react-native-paper": "^4.12.4",
     "react-native-safe-area-context": "4.3.1",
```

Comment:
@HunnySajid could you change this to "react-native-safe-area-context": "^4.2.4", to avoid the safe area gifted chat bug that I think you got past with npm dedupe.

Reply:
sure